docs/components/clerk-provider.mdx

```diff
@@ -204,7 +204,7 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
   - `signInUrl`
   - `string`
 
-  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances and required for development instances. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
+  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. It's required for development instances if you a use satellite application. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```

Comment:
```suggestion
  This URL will be used for any redirects that might happen and needs to point to your primary application on the client-side. This option is optional for production instances. It's required for development instances if you use a satellite application. It's recommended to use [the environment variable](/docs/deployments/clerk-environment-variables#sign-in-and-sign-up-redirects) instead.
```

Reply:
Is it required to be set in the satellite application, or the primary application ?

Reply:
In the satellite application